server/index.js

```diff
@@ -30,7 +30,7 @@ app.set('port', port);
 
 // Import and Set Nuxt.js options
 let config = require('../nuxt.config.js');
-config.dev = !(process.env.NODE_ENV === 'production');
+config.dev = !(process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'test');
```

Comment:
What does this aim to achieve?

Reply:
Run the test server(localhost:1337) without development mode so that it doesn't display linting errors.